java-client/build.gradle.kts

```diff
@@ -126,6 +126,7 @@ dependencies {
     // Apache 2.0
     implementation("com.fasterxml.jackson.core", "jackson-core", jacksonVersion)
     implementation("com.fasterxml.jackson.core", "jackson-databind", jacksonVersion)
+    testImplementation("com.fasterxml.jackson.datatype", "jackson-datatype-jakarta-jsonp", jacksonVersion)
```

Comment:
Should this be `testImplementation` or `implementation`?

Reply:
Thanks for looking, @mitalawachat. By and large, this module is optional: the application may not manipulate `JsonObject`s or/and use different provider, fe `JsonbJsonpMapper`. It is used in tests but making it required is not necessary.